PlayBook/src/getting_started/download_playcover.md

```diff
@@ -2,22 +2,18 @@
 
 ## Prerequisites
 
-At the moment, PlayCover can only run on Apple Silicon Macs. Devices with the following chips are supported:
-
-* M1
-* M1 Pro
-* M1 Max
-* M1 Ultra
-* M2
+At the moment, PlayCover only works on **Apple Silicon Macs**. The minimum hardware requirement is an Apple M1 chip, PlayCover will also work on all Apple Silicon chips released after the M1.
 
 If you have an Intel Mac, you can explore alternatives like Bootcamp or emulators.
 
 ## Download
 
-You can download stable releases [here](https://github.com/PlayCover/PlayCover/releases), or build from source by following the instructions below.
+You can download the [latest stable release](https://github.com/PlayCover/PlayCover/releases/latest), or build from source by following the instructions below. 
```

Comment:
This ‘below’ seems to be unintended

Reply:
it's way below the page i think lol

Reply:
They aren’t since it seems to refer to an old version of the document. Building instructions have since been moved to another article.

Reply:
Should i just make it link to this page then? https://docs.playcover.io/building_from_source/install_prerequisites

Reply:
i thought the homebrew cask thing was building from source